contents/handbook/small-teams/feature-success/objectives.mdx

```diff
@@ -1,32 +1,29 @@
 As always, reliability is the #1 unwritten goal: Making sure feature flags are reliable trumps every other objective.
 
 
-### Objective: Make it easier for engineers to analyze the success of a feature
+### Objective: Get 5 reference customers for surveys
 
-Last quarter, we built the foundation for feature success: All sorts of different tools that people can use to help determine success.
+Last quarter, we built out surveys which has been very well received. This quarter, we're doubling down on surveys.
 
-This quarter is about bringing it all together and making it seamless to connect these features together.
+We will
 
-This will involve a few things:
-  - Validate every component is useful on its own (flag autocapture, surveys, dashboard templates on flags, and early access management).
-  - Integrate every pair of components together: i.e. validate every pair of the above components can multiply usefulness over a single component
-  - Integrate all components together in one place: A notebook template that brings feature success together.
+- Launch pricing for surveys, make it its own standalone product.
+- Build out features critical to paying users, like templates, better result visualisation, data export, and potentially event targeting for surveys.
```

Comment:
Build out features critical to surveys like templates, multiple questions and logic, better result visualizations, data export, and event targeting.